auto_joiner.py

```diff
@@ -667,6 +687,7 @@ def main():
                 if meeting_to_join is not None:
                     total_members = 0
                     join_meeting(meeting_to_join)
+                    # TODO:
```

Comment:
Why a todo here? Joining a meeting already has a notification

Reply:
Actually i wrote all the todos, i think at last i forgot to remove this. Ill do it